packaging/nnstreamer.spec

```diff
@@ -102,7 +104,22 @@ pushd build
 popd
 
 pushd tests
-./testAll.sh
+timeout=600
+./testAll.sh &
+pid=$!
+while ((timeout > 0));
+do
+    sleep 1
+    if [[ ! $(ps | grep "$pid") ]]; then
+        break;
+    fi
+    (( timeout -= 1))
+done
+if [[ $(ps | grep "$pid") ]]; then
+    kill $pid
+    echo "GBS is stopped because timeout(5min) of './testAll.sh'"
+    exit 124
```

Comment:
Please append a comment on a meaning of the return value 124.

Reply:
fixed. 
link: https://unix.stackexchange.com/questions/205076/timeout-function-return-value